content/blog/2016/2016-09-27-jenkins-world-2016-wrap-up-scaling.adoc

```diff
@@ -0,0 +1,141 @@
+---
+layout: post
+title: "Jenkins World 2016 Wrap-up - Scaling"
+tags:
+- event
+- jenkinsworld
+author: lnewman
+---
+
+NOTE: This is a guest post by link:https://github.com/bitwiseman[Liam Newman],
+Technical Evangelist at CloudBees.
+
+image:/images/conferences/Jenkins-World_125x125.png[Jenkins World 2016, role=right]
+
+I've used Jenkins for several years and been a contributor for the last two, but
+this was my first time at a Jenkins user conference.
+
+NOTE: More lead-in goes here, I'll get back to this.
+
+As I noted before, link:http://cloudbees.com[CloudBees] recorded all the sessions and will
+eventually post them (editing takes time).  I'll post an update when they become available.
+In the meanwhile, here's a recap of the some of the sessions related to scaling Jenkins,
+with links to slides:
+
+---
+
+image:/images/post-images/scaling-jenkins-at-jenkinsworld/159px-National_Public_Radio_logo.svg.png[role=right]
+
+
+link:https://twitter.com/milespj[Paul Miles] and
+link:https://twitter.com/jgrantd[Grant Dickie] of
+link:https://www.npr.org/[NPR] talked about
+link:https://www.cloudbees.com/jenkinsops-initiative-streamline-and-automate-jenkins[*JenkinsOps: An Initiative to Streamline and Automate Jenkins*].
+They shared ways their team has used Jenkins to automate many of the
+administrative tasks related to managing feature code branches,
+handling deployments, running tests, and properly configuring their environments.
+They also showed code samples and talked about future challenges in their quest
+to achieve continuous deployment.
```

Comment:
I suggest linking to Wikipedia or elsewhere on "continuous deployment" to help differentiate between it and "continuous delivery"